### PR TITLE
PPF-300 - Add setup playwright

### PR DESCRIPTION
**Pull request by simon-debruijn**
### Added
- Add playwright setup with basic auth

---
Ticket: https://jira.uitdatabank.be/browse/PPF-300

```
Running 1 test using 1 worker
  1 passed (9.9s)
```

Related appconfig PR:
https://github.com/cultuurnet/appconfig/pull/577

### CODE REVIEW

File: playwright.config.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: "./e2e",
+  fullyParallel: true,
+  forbidOnly: !!process.env.CI,
+  retries: process.env.CI ? 2 : 0,
+  workers: process.env.CI ? 1 : undefined,
+  reporter: "html",
```
Comment (simon-debruijn) [inline playwright.config.ts:14]: Kristof mentioned junit output can be formatted nicely. SO we could change the output if needed later

**Comment by simon-debruijn** [comment]
You can run the tests by doing `npx playwright test` in the root of the project. I'll add scripts for it in package.json and the makefile

**Comment by simon-debruijn** [comment]
I'm using the credentials from `Dev End2End Test ACC` that are stored in bitwarden

**Review by LucWollants — COMMENTED** [review]
A few questions/remarks:
- Can it also be added as a `make` command to the `Makefile`?
- Not sure if it needs to be added to the docs?
- It also requires a change to the `.ENV` file?
- Should the `baseURL` be configurable?

**Comment by simon-debruijn** [comment]
> A few questions/remarks:
> 
> * Can it also be added as a `make` command to the `Makefile`?
> * Not sure if it needs to be added to the docs?
> * It also requires a change to the `.ENV` file?
> * Should the `baseURL` be configurable?

I've addressed all your remarks

File: README.md [hunk 2]
```diff
@@ -221,12 +221,32 @@ $ make stan
...
+```
+$ make test-e2e
+```
+
+- Run an e2e test filtered on file path or parts of file name
+```
+$ make test-e2e-filter filter=view-integrations-in-overview.test.ts
+```
+
+- Run an e2e filtered on the test title
```
Comment (simon-debruijn) [inline README.md:249]:
```suggestion
- Run an e2e test filtered on the test title
```

**Pull request merged**